src/Common.am

```diff
@@ -73,3 +73,15 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+EXTENSIONS=.hdrtest
+.h.hdrtest:
```

Comment:
Should we add a dependency on testHeaders.cc.in to this target?

Reply:
That would be the logic step, but external dependencies in extension-based targets are not allowed